SonarQube.Common/AnalysisProperties/Property.cs

```diff
@@ -72,6 +77,11 @@ public static bool IsValidKey(string key)
             return isValid;
         }
 
+        public static bool AreKeysEqual(string key1, string key2)
+        {
+            return PropertyKeyComparer.Equals(key1, key2);
+        }
+
```

Comment:
Miscellaneous: I added a public method rather than exposing the comparer directly.